CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing guidelines
+
+<!-- vim-markdown-toc GFM -->
+
+- [Development](#development)
+  - [System prerequisites](#system-prerequisites)
+  - [Dev environment set up](#dev-environment-set-up)
+- [Pull requests](#pull-requests)
+- [Merging to main](#merging-to-main)
+  - [Stages: Draft and Ready for review](#stages-draft-and-ready-for-review)
+
+<!-- vim-markdown-toc -->
+
+## Development
+
+### System prerequisites
+
+This project requires:
+
+- [docker](https://docs.docker.com/get-docker/)
+- [nodejs LTS (18)](https://nodejs.org/en/download/)
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+Optional:
+
+- [Use a node version manager](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-version-manager-to-install-nodejs-and-npm)
+
+### Dev environment set up
+
+```bash
+# Create an .env based on the example
+$ cp .env.example .env
+
+# Start MySQL in docker
+$ docker compose up -d
+# Wait for the db to finish starting, it takes a few seconds before it's ready...
+
+# Install all dependencies
+$ npm install
+
+# Setup the database
+$ npm run setup
+
+# Generate a build folder the first time running the app
+$ npm run build
```

Comment:
I saw an issue go by today about this from @dadolhay, we should maybe clarify the other of this.

Reply:
#60? yeah we noticed it during our debugging session after the main meeting today. We said we don't know how to exactly fix it. So we are gonna make a note in the docs for now, until someone finds a fix for it.

Reply:
OK, great.